### PR TITLE
Rename snake_case functions in OpenRCT2/util folder

### PR DESCRIPTION
**Pull request by Broxzier**

### CODE REVIEW

File: src/openrct2/util/Util.h [hunk 1]
```diff
@@ -18,42 +18,42 @@
...
 std::vector<uint8_t> Ungzip(const void* data, const size_t dataLen);
 
-int8_t add_clamp_int8_t(int8_t value, int8_t value_to_add);
-int16_t add_clamp_int16_t(int16_t value, int16_t value_to_add);
-int32_t add_clamp_int32_t(int32_t value, int32_t value_to_add);
-int64_t add_clamp_int64_t(int64_t value, int64_t value_to_add);
-money16 add_clamp_money16(money16 value, money16 value_to_add);
-money32 add_clamp_money32(money32 value, money32 value_to_add);
-money64 add_clamp_money64(money64 value, money64 value_to_add);
+int8_t AddClamp_int8_t(int8_t value, int8_t value_to_add);
```
Comment (duncanspumpkin) [inline src/openrct2/util/Util.h:44]: I thought this was a bit odd but its probably the best that can be done without making it a template.

File: src/openrct2/util/Util.cpp [hunk 2]
```diff
@@ -206,12 +206,12 @@ static int32_t bitcount_lut(uint32_t source)
 
 static int32_t (*bitcount_fn)(uint32_t);
```
Comment (duncanspumpkin) [inline src/openrct2/util/Util.cpp:207]: This function pointer should be updated as well to `BitcountFn`

**Review by duncanspumpkin — APPROVED** [review]
Only thing is that function pointer.

File: src/openrct2/drawing/LightFX.cpp [hunk 3]
```diff
@@ -855,9 +855,9 @@ void LightFXApplyPaletteFilter(uint8_t i, uint8_t* r, uint8_t* g, uint8_t* b)
     float sunLight = std::max(0.0f, std::min(1.0f, 2.0f - night * 3.0f));
 
     // Night version
-    natLightR = flerp(natLightR * 4.0f, 0.635f, (std::pow(night, 0.035f + sunLight * 10.50f)));
-    natLightG = flerp(natLightG * 4.0f, 0.650f, (std::pow(night, 0.100f + sunLight * 5.50f)));
-    natLightB = flerp(natLightB * 4.0f, 0.850f, (std::pow(night, 0.200f + sunLight * 1.5f)));
+    natLightR = FLerp(natLightR * 4.0f, 0.635f, (std::pow(night, 0.035f + sunLight * 10.50f)));
```
Comment (Gymnasiast) [inline src/openrct2/drawing/LightFX.cpp:858]: Is the FLerp casing deliberate?

File: src/openrct2/interface/Chat.cpp [hunk 4]
```diff
@@ -216,7 +216,7 @@ void ChatAddHistory(std::string_view s)
     time(&timer);
     auto tmInfo = localtime(&timer);
     char timeBuffer[64]{};
-    strcatftime(timeBuffer, sizeof(timeBuffer), "[%H:%M] ", tmInfo);
+    StrCatfTime(timeBuffer, sizeof(timeBuffer), "[%H:%M] ", tmInfo);
```
Comment (Gymnasiast) [inline src/openrct2/interface/Chat.cpp:219]: StrCatFTime - the f stands for format.

File: src/openrct2/util/SawyerCoding.cpp [hunk 5]
```diff
@@ -18,14 +18,14 @@
 #include <cstring>
 #include <stdexcept>
 
-static size_t decode_chunk_rle(const uint8_t* src_buffer, uint8_t* dst_buffer, size_t length);
-static size_t decode_chunk_rle_with_size(const uint8_t* src_buffer, uint8_t* dst_buffer, size_t length, size_t dstSize);
+static size_t DecodeChunkRLE(const uint8_t* src_buffer, uint8_t* dst_buffer, size_t length);
+static size_t DecodeChunkRLEWithWize(const uint8_t* src_buffer, uint8_t* dst_buffer, size_t length, size_t dstSize);
```
Comment (Gymnasiast) [inline src/openrct2/util/SawyerCoding.cpp:22]: Typo

File: src/openrct2/util/Util.cpp [hunk 6]
```diff
@@ -191,7 +191,7 @@ static int32_t bitcount_popcnt(uint32_t source)
 #endif
 }
 
-static int32_t bitcount_lut(uint32_t source)
+static int32_t BitcountLut(uint32_t source)
```
Comment (Gymnasiast) [inline src/openrct2/util/Util.cpp:194]: Inconsistent spelling - you’re using both `Bitcount` and `BitCount`.

File: src/openrct2/drawing/LightFX.cpp (hunk 3 above)
Comment (Broxzier) [reply src/openrct2/drawing/LightFX.cpp:858]: Yes, it's a lerp function for floats, hence the F.

File: src/openrct2/util/Util.cpp [hunk 7]
```diff
@@ -161,20 +161,20 @@ bool avx2_available()
     return false;
 }
 
-static bool bitcount_popcnt_available()
+static bool BitcountPopcntAvailable()
```
Comment (Gymnasiast) [inline src/openrct2/util/Util.cpp:164]: This one is still not changed.

**Review by Gymnasiast — APPROVED** [review]

**Comment by Gymnasiast** [comment]
Ik zie alles xD

**Pull request merged**